octree_web_viewer/src/backend.rs

```diff
@@ -75,106 +74,103 @@ fn pad(input: &mut Vec<u8>) {
     }
 }
 
-pub struct NodesData {
-    octree: Arc<Octree>,
+fn get_octree_from_state(
+    octree_id: String,
```

Comment:
Can we pass a `&str` or `AsRef<>` instead?

Reply:
done